controller/invitation.go

```diff
@@ -0,0 +1,80 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-auth/app"
+	"github.com/fabric8-services/fabric8-auth/application"
+	"github.com/fabric8-services/fabric8-auth/authorization/invitation"
+	invitationService "github.com/fabric8-services/fabric8-auth/authorization/invitation/service"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/jsonapi"
+	"github.com/fabric8-services/fabric8-auth/log"
+	"github.com/fabric8-services/fabric8-auth/login"
+	"github.com/goadesign/goa"
+	"github.com/satori/go.uuid"
+)
+
+// InvitationController implements the invitation resource.
+type InvitationController struct {
+	*goa.Controller
+	db         application.DB
+	invService invitationService.InvitationService
+}
+
+// NewInvitationController creates a invitation controller.
+func NewInvitationController(service *goa.Service, db application.DB, invitationService invitationService.InvitationService) *InvitationController {
+	return &InvitationController{Controller: service.NewController("InvitationController"), db: db, invService: invitationService}
+}
+
+// Create runs the create action.
+func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationContext) error {
+	currentIdentity, err := login.LoadContextIdentityIfNotDeprovisioned(ctx, c.db)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
```

Comment:
Don't need to wrap the error to errors.NewUnauthorizedError(err.Error())). 
login.LoadContextIdentityIfNotDeprovisioned(ctx, c.db) returns UnauthorizedError.
So, return jsonapi.JSONErrorResponse(ctx, err) is enough here.

Reply:
Fixed